tools/notification-configuration/github-codeowner-subscriber/Program.cs

```diff
@@ -137,7 +128,7 @@ class Program
 
                         // Get set of team members in the CODEOWNERS file
                         var contactResolutionTasks = codeOwnerEntry.Owners
-                            .Select(contact => githubNameResolver.GetInternalUserPrincipal(contact));
+                            .Select(contact => githubToAadAliasResolver.GetMsAliasFromGithub(contact) + "@microsoft.com");
```

Comment:
I believe some aliases may be `ntdev.microsoft.com` but I'm not sure. I think assuming `@microsoft.com` may be the best we can do though.

Reply:
I found we have the `userNamePrinciple` which is employee email. Use it directly